dao/src/main/scala/za/co/absa/enceladus/dao/MenasRestDAO.scala

```diff
@@ -116,10 +113,9 @@ object MenasRestDAO extends MenasDAO {
     sendPostJson(url, json)
   }
 
-  private def sendPostJson(url: String, json: String): Boolean = {
```

Comment:
Sorry, I'm being nitpicky. But consider this. instead of passing "retryCount" as a "current number of retries" you can pass "maxRetry". And recursively call `sendPostJson(url, json, retryCount - 1)` if `retryCount >0`. This way you won't need a `maxRetries` constant and you let the client choose the number of retries. A reasonable default would still be `1`.